CinemaCat.Api/DTO/CreateMovieModel.cs

```diff
@@ -2,4 +2,4 @@
 
 namespace CinemaCat.Api.DTO;
 
-public record class CreateMovieModel(string Title, string ReleasedDate, Person Director, int Rating);
\ No newline at end of file
+public record class CreateMovieModel(string Title, string ReleasedDate, string DirectorName, int Rating);
```

Comment:
let's leave it Person with opitonal GUID
a list of TOP actors can be also added here